quantum/stm32/chconf.h

```diff
@@ -58,7 +58,7 @@
  *          The value one is not valid, timeouts are rounded up to
  *          this value.
  */
-#    define CH_CFG_ST_TIMEDELTA 2
+#    define CH_CFG_ST_TIMEDELTA	0
```

Comment:
use space instead of tab

Reply:
Ideally instead this should be run through clang-format instead